ci/2.7.pylintrc

```diff
@@ -7,19 +7,12 @@
 # bad-open-mode<W1501>: (hi-pri) Python supports: r, w, a[, x] modes with b, +, and U (only with r) options.
 # bad-super-call<E1003>: (hi-pri) Used when another argument than the current class is given as first argument of the super builtin.
 # bare-except<W0702>: (hi-pri) Used when an except clause doesn't specify exceptions type to catch.
-# blacklisted-name<C0102>: (needs review) Used when the name is listed in the black list (unauthorized names).
 # cell-var-from-loop<W0640>: (hi-pri) A variable used in a closure is defined in a loop. This will result in all closures using the same value for the closed-over variable.
-# cyclic-import<R0401>: (hi-pri) Used when a cyclic import between two or more modules is detected.
 # expression-not-assigned<W0106>: (hi-pri) Used when an expression that is not a function call is assigned to nothing.
-# duplicate-code<R0801>: (hi-pri) Indicates that a set of similar lines has been detected among multiple file. (also disabled globally)
 # function-redefined<E0102>: (hi-pri) Used when a function / class / method is redefined.
 # global-statement<W0603>: Used when you use the "global" statement to update a global variable.
 # import-error<E0401>: (hi-pri) Used when pylint has been unable to import a module.
-# inconsistent-return-statements<R1710>: (hi-pri) if any return statement returns an expression, any return statements where no value is returned should explicitly state this as return None, and an explicit return statement should be present at the end of the function
```

Comment:
Maybe this too?

Reply:
Instead of suppressing all R warning, I changed the settings to suppress only the warnings the agent is disabling. I added this one back.

Reply:
Awesome thanks! If we're not blocking this then we should keep this comment too

Reply:
The original intention of those comments was to document the text description of the message, but they are already out of sync with the code and are likely to get more out of sync as we make additional changes. I was inclined to remove them altogether, but the rcfile for 2.7 has some infor wrt python versions that may be useful.